src/main/java/dev/lambdaurora/lambdabettergrass/mixin/ModelLoaderMixin.java

```diff
@@ -78,15 +79,13 @@ private void onPutModel(Identifier id, UnbakedModel unbakedModel, CallbackInfo c
 				// Find and load states metadata if not cached.
 				if (state == null) {
 					var stateResourceId = new Identifier(stateId.getNamespace(), stateId.getPath() + ".json");
-					if (this.resourceManager.containsResource(stateResourceId)) {
-						try {
-							var json = JsonParser.parseReader(
-									new InputStreamReader(this.resourceManager.method_14486(stateResourceId).getInputStream())
-							).getAsJsonObject();
-							state = LBGState.getOrLoadMetadataState(stateId, this.resourceManager, json, this.variantMapDeserializationContext);
-						} catch (IOException e) {
-							// Ignore.
-						}
+					try (var reader = new InputStreamReader(resourceManager.getResourceOrThrow(stateResourceId).open())) {
+						var json = JsonParser.parseReader(reader).getAsJsonObject();
+						state = LBGState.getOrLoadMetadataState(stateId, this.resourceManager, json, this.variantMapDeserializationContext);
+					} catch (FileNotFoundException e) {
+						// Ignore.
+					} catch (IOException e) {
+						// Ignore.
```

Comment:
```suggestion
					} catch (IOException e) {
						// Ignore.
```

FileNotFoundException extends IOException.